app/core/settings.py

```diff
@@ -88,6 +92,55 @@
     'DATETIME_FORMAT': "%Y-%m-%dT%H:%M:%SZ",
 }
 
+LOG_ATTRS = ['asctime', 'created', 'levelname', 'message',
+             'filename', 'funcName', 'lineno', 'module', 'pathname']
+LOG_FORMAT_STR = ' '.join(['%(' + v + ')s' for v in LOG_ATTRS])
+
+LOGGING = {
+    'version': 1,
+    'disable_existing_loggers': False,
+    'formatters': {
+        'json': {
+            '()': jsonlogger.JsonFormatter,
+            'format': LOG_FORMAT_STR,
+        },
+        'verbose': {
+            'format': "[%(asctime)s] %(levelname)s [%(name)s:%(lineno)s] %(message)s",
+            'datefmt': "%d/%b/%Y %H:%M:%S"
```

Comment:
this date format is a pain, wort it to change to `%Y-%b-%dT%H:%M:%SZ` (assuming UTC)